### PR TITLE
Task 18 photo gallery

### PR DESCRIPTION
**Pull request by helly-15**
Added Photo Gallery
Task https://trello.com/c/flT79QEs/9-task-18photo-gallery

### CODE REVIEW

File: src/components/AuthorPage.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
-import React from 'react';
+import React, {Fragment} from 'react';
```
Comment (vvsvintsitsky) [inline src/components/AuthorPage.js]: Please, use [shorthand](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html) for fragments.

File: src/components/author-information/image-gallery/ImageGallery.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, {Fragment} from 'react';
+import ImageGallery from 'react-image-gallery';
+import data from '../../../data/data.json';
+import './image-gallery.css';
+import Typography from "@material-ui/core/Typography";
+import {Grid} from "@material-ui/core";
+import {useParams} from "react-router-dom";
+import PropTypes from "prop-types";
+const imageSet = (id=0)=>{
```
Comment (vvsvintsitsky) [inline src/components/author-information/image-gallery/ImageGallery.jsx]: `const getImageSet = photographer => photographer.gallery.map(({ link }) => ({ original: link, thumbnail: link }));`

File: src/components/author-information/image-gallery/ImageGallery.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    let images= data.photographers[id].gallery.map((work)=>{
```
Comment (vvsvintsitsky) [inline src/components/author-information/image-gallery/ImageGallery.jsx]: Hardcoded use of _data_ object should be removed.

**Comment by slolkunchik** [comment]
approved

**Pull request merged**